### PR TITLE
Feature/e UI 5543 hearing attendance page model changes

### PR DESCRIPTION
**Pull request by kuda1992**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-5543?filter=-1

### Change description ###
hearing-attendance page model changes

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/hearings/containers/request-hearing/hearing-attendance/hearing-attendance.component.ts [hunk 1]
```diff
@@ -138,12 +137,12 @@ export class HearingAttendanceComponent extends RequestHearingPageFlow implement
   }
 
   public patchValues(party: PartyDetailsModel): FormGroup {
+    console.log('in patch values', party);
```
Comment (johnbenjamin-hmcts) [inline src/hearings/containers/request-hearing/hearing-attendance/hearing-attendance.component.ts:140]: console.log statement

File: src/hearings/hearing.test.data.ts [hunk 2]
```diff
@@ -988,10 +988,10 @@ export const serviceHearingValuesModel: ServiceHearingValuesModel = {
     locationId: '196538',
     locationType: 'hearing',
   },
-  {
-    locationId: '234850',
-    locationType: 'hearing',
-  },
+    {
```
Comment (johnbenjamin-hmcts) [inline src/hearings/hearing.test.data.ts:991]: Can you correct the alignment

**Review by johnbenjamin-hmcts — APPROVED** [review]
Approved with minor comments

File: api/hearings/data/hearingRequests.mock.data.ts [hunk 3]
```diff
@@ -76,7 +76,6 @@ export const HEARING_REQUEST_RESULTS: HearingRequestMainModel[] = [
         partyID: 'P1',
         partyName: 'Jane and Smith',
         partyType: PartyType.IND,
-        partyChannel: 'inPerson',
```
Comment (rayliangatos) [inline api/hearings/data/hearingRequests.mock.data.ts:79]: in this mock data, after the partyChannel: 'inPerson', is removed here, could this be added to the the individualDetails.preferredChannel.

File: api/hearings/data/hearingRequests.mock.data.ts [hunk 4]
```diff
@@ -104,7 +103,6 @@ export const HEARING_REQUEST_RESULTS: HearingRequestMainModel[] = [
         partyID: 'P2',
         partyName: 'DWP',
         partyType: PartyType.ORG,
-        partyChannel: 'byVideo',
```
Comment (rayliangatos) [inline api/hearings/data/hearingRequests.mock.data.ts:107]: in this mock data, after the partyChannel: 'inPerson', is removed here, could this be added to the the individualDetails.preferredChannel.

File: api/hearings/data/hearingRequests.mock.data.ts [hunk 5]
```diff
@@ -150,7 +148,6 @@ export const HEARING_REQUEST_RESULTS: HearingRequestMainModel[] = [
             partyName: 'Jane and Smith',
             partyType: PartyType.IND,
             partyRole: 'appellant',
-            partyChannel: 'inPerson',
```
Comment (rayliangatos) [inline api/hearings/data/hearingRequests.mock.data.ts:153]: in this mock data, after the partyChannel: 'inPerson', is removed here, could this be added to the the individualDetails.preferredChannel.

File: api/hearings/data/hearingRequests.mock.data.ts [hunk 6]
```diff
@@ -178,7 +175,6 @@ export const HEARING_REQUEST_RESULTS: HearingRequestMainModel[] = [
             partyName: 'DWP',
             partyType: PartyType.ORG,
             partyRole: 'claimant',
-            partyChannel: 'byVideo',
```
Comment (rayliangatos) [inline api/hearings/data/hearingRequests.mock.data.ts:181]: and same for all others below

File: api/hearings/data/serviceHearingValues.mock.data.ts [hunk 7]
```diff
@@ -67,7 +67,6 @@ export const SERVICE_HEARING_VALUES: ServiceHearingValuesModel = {
       partyID: 'P1',
       partyType: PartyType.IND,
       partyName: 'Jane and Smith',
-      partyChannel: 'byVideo',
```
Comment (rayliangatos) [inline api/hearings/data/serviceHearingValues.mock.data.ts:70]: in this mock data, after the partyChannel is removed here, could this be added to the the individualDetails.preferredChannel, this needs replacement not removed entirely.

**Review by rayliangatos — APPROVED** [review]
thanks looks good to me

**Pull request merged**